libultraship/libultraship/Lib/Fast3D/gfx_pc.cpp

```diff
@@ -2030,12 +2031,11 @@ static void gfx_s2dex_bg_copy(const uObjBg* bg) {
 static inline void* seg_addr(uintptr_t w1)
 {
     // Segmented?
-    if (w1 >= 0xF0000000)
+    if (w1 & 1)
```

Comment:
Were these constants initially incorrect? Might it be better to have a `#define` (or at least comment) indicating what these are supposed to represent?